apps/golden-sample-app/src/app/transactions/transaction-additional-details.component.html

```diff
@@ -0,0 +1,9 @@
+<span class="transaction-account-number">{{
+  context?.counterPartyAccountNumber
+}}</span>
+<span
+  class="transaction-my-addition"
+  *ngIf="context?.additions?.['myAddition']"
+>
+  My Addition: {{ context?.additions?.['myAddition'] }}
```

Comment:
Shall we advice this way of accessing data from inner levels? I would suggest handling it in the component code. This way it can look cleaner and we can demo setting defaults as well.

Reply:
You mean the safe dereferencing op?